stitch_comp/CfgDataStitch.py

```diff
@@ -0,0 +1,165 @@
+#!/usr/bin/env python
+
+## @ CfgDataStitch.py
+# Project template script to patch CFGDATA blob into IFWI image
+#
+# Copyright (c) 2018, Intel Corporation. All rights reserved.<BR>
+# SPDX-License-Identifier: BSD-2-Clause-Patent
+#
+##
+
+import os
+import sys
+import argparse
+import subprocess
+
+dlt_files = ['CfgData_Default.dlt', 'CfgData_Int_LeafHill.dlt', 'CfgData_Ext_Gpmrb.dlt', 'CfgData_Ext_Up2.dlt', 'CfgData_Ext_OxbHill.dlt', 'CfgData_Ext_MB3.dlt', 'CfgData_Ext_JuniperHill.dlt']
```

Comment:
this patch creates a new folder stitch_comp for this script, looks this script is  generic for all the platforms. but from the dlt file, looks it is APL specific.

Reply:
Good catch!  That was generated files by stitching.  Since it was not in the .gitignore,  it was added by mistake.  Will remove it.